src/components/Fields/TextArea.scss

```diff
@@ -107,9 +87,41 @@ $textFieldLabelColor: $charcoal !default;
         width: 100%;
     }
 
+    &-validation-message {
+        margin-top: 8px;
+        font-family: "DINOT", Arial, sans-serif;
+        font-size: 11px;
+        font-weight: normal;
+        color: $pink2;
+        letter-spacing: .92px;
+    }
+
     .uir-icon-required {
         width: 10px;
         height: 10px;
         margin-left: 5px;
     }
 }
+
+.uir-text-area-inner {
```

Comment:
Could this be a breaking change if any clients are overriding these styles?

Reply:
You mean they're using this same class name? I had to use it because it has to be applied to the innermost div to draw the pink underline under the field due to the addition of the external `div`. What are your suggestions for it?